### PR TITLE
Add new authorization-control configurations.

### PR DESCRIPTION
**Pull request by Thisara-Welmilla**
Add new config for authorization configurations.

### CODE REVIEW

File: features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json [hunk 1]
```diff
@@ -222,6 +222,9 @@
   "account_recovery.endpoint.auth.name":"dashboard",
   "account_recovery.endpoint.auth.hash":"66cd9688a2ae068244ea01e70f0e230f5623b7fa4cdecb65070a09ec06452262",
 
+  "authorization_control.skip_authorization.client_authentication.name":"ClientAuthentication",
```
Comment (sachiniWettasinghe) [inline features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json:225]: shouldn't this be authorization_control.skip_authorization.authenticator.name?

File: features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json [hunk 2]
```diff
@@ -222,6 +222,9 @@
...
+  "authorization_control.skip_authorization.client_authentication.allowedEndpoints": "(.*)/accountrecoveryendpoint(.*),(.*)/api/identity/recovery/(.*),(.*)/data/AuthRequestKey/(.*),(.*)/applications(.*),(.*)/identity-governance(.*),(.*)/user/(.*)/validate-username(.*),(.*)/consent-mgt/v1.0/consents/purposes(.*),(.*)/user/v1.0/me(.*)",
```
Comment (sachiniWettasinghe) [inline features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json:226]: shouldn't this be authorization_control.skip_authorization.authenticator.allowedEndpoints ?

File: features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json (hunk 1 above)
Comment (Thisara-Welmilla) [reply features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json:225]: `authorization_control.skip_authorization` is the common part and next part need to be the authenticator name to support to configure any number of authenticators from xml.j2 template.

File: features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json (hunk 2 above)
Comment (Thisara-Welmilla) [reply features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json:226]: `authorization_control.skip_authorization` is the common part and next part need to be the authenticator name to support to configure any number of authenticators from xml.j2 template.

**Review by sachiniWettasinghe — DISMISSED** [review]

**Review by sachiniWettasinghe — APPROVED** [review]

**Pull request merged**